pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+###############################################################################
+[tool.poetry]
+name = "smqtk_image_io"
+version = "0.15.0"
```

Comment:
It's OK for now that  you've bumped the version w.r.t. SMQTK mono-repo, but remember that this will be the version that our first release will be for this package.